sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1410,7 +1410,7 @@ def __next__(self):
 
 
 @deprecated(since='2.11.0', current="WriteToBigQuery")
-class BigQuerySink(*args, **kwargs):
+def BigQuerySink(*args, **kwargs):
```

Comment:
Why change this from class to def?

Reply:
This way one can still do `io.Write(BigQuerySink(...))` even though the old class is gone.